app/Controllers/categoryController.php

```diff
@@ -41,18 +41,18 @@ public function createAction() {
 			invalidateHttpCache();
 
 			$cat_name = trim(Minz_Request::param('new-category', ''));
-			if ($cat_name == '') {
+			if ($cat_name === '') {
 				Minz_Request::bad(_t('feedback.sub.category.no_name'), $url_redirect);
 			}
 
 			$cat = new FreshRSS_Category($cat_name);
 
-			if ($catDAO->searchByName($cat->name()) != null) {
+			if ($catDAO->searchByName($cat->name()) !== null) {
 				Minz_Request::bad(_t('feedback.sub.category.name_exists'), $url_redirect);
 			}
 
 			$opml_url = checkUrl(Minz_Request::param('opml_url', ''));
-			if ($opml_url != '') {
+			if ($opml_url !== '') {
```

Comment:
No, `checkUrl` might return `false`.
Please don't blindly change operators. Most have been chosen for a reason ;-)